esmvalcore/preprocessor/_io.py

```diff
@@ -83,10 +86,18 @@ def load(
     file:
         File to be loaded. If ``file`` is already a loaded dataset, return it
         as a :class:`~iris.cube.CubeList`.
+        File as ``Path`` object could be a Zarr store.
     ignore_warnings:
         Keyword arguments passed to :func:`warnings.filterwarnings` used to
         ignore warnings issued by :func:`iris.load_raw`. Each list element
         corresponds to one call to :func:`warnings.filterwarnings`.
+    backend_kwargs:
```

Comment:
You should mention that this is currently only used for loading Zarr files (we might want to expand this later).

Reply:
b5c3301ca0bab256c580556ef9cc3f8fddc3ecd8